### DISCUSSION TITLE
Adding more stuffs in UI Framework, Data Provider, Live Providers, Auth Providers

### DISCUSSION THREAD

**Discussion by AriaFantom** [refinedev/refine#5491] [Integration request] [answered]
### These are the Integration which i wanna see in future of refine:

1. **UI Lib: Shadcn, NextUI.**
2. **Data/Live Provider: PocketBase.**
3. **Auth Providers: Next-Auth/Auth.js**

### Why I requested:

- NextAuth is free and easy auth library for anyone who is new to authentication and it's also support typescript and have adapters.
- PocketBase is very new and not to be used in high load production based software but it is sufficient for normal developers or small projects.
- Recently Shadcn has become very popular and it also goes well with nextUI library both can have a very good combination of 2 UI libraries.

**Comment by BatuhanW** [comment, marked as answer]
Hello @AriaFantom, thanks for the suggestions! We are open to contributions for any integration from the community.

We already have NextAuth example, you can see it here: https://refine.dev/docs/examples/next-js/NextAuth-js/
